doc/cephadm/services/mgmt-gateway.rst

```diff
@@ -0,0 +1,136 @@
+.. _deploy-cephadm-mgmt-gateway:
+
+=============
+Mgmt Gateway
```

Comment:
I'll defer to @zdover23 but I think this heading should say Management Gateway

Reply:
fixed